webpack/published-converted-components.js

```diff
@@ -0,0 +1,19 @@
+const path = require("path");
```

Comment:
No need for 2 separate files for published components, export 2 constants from the existing `published-components` file - `publishedComponents` and `publishedTSComponents`.